rewriter/SourceRewriter.cpp

```diff
@@ -726,8 +726,10 @@ class FnDecl : public RefactoringCallback {
     assert(result.SourceManager != nullptr);
     auto &sm = *result.SourceManager;
 
+    Function fn_name = fn_node->getNameAsString();
+
     // Ignore declarations in libc and libia2 headers
-    if (ignore_file(get_filename(fn_node->getLocation(), sm))) {
+    if (ignore_file(get_filename(fn_node->getLocation(), sm)) && !fn_name.starts_with("late_init_tls_")) {
```

Comment:
Is this leftover? I don't see this symbol being defined in this PR

Reply:
It is, I'll remove this and squash some other stuff to simplify the history.